solana/program_v2/Anchor.toml

```diff
@@ -0,0 +1,23 @@
+[features]
+seeds = false
+
+#TODO: Updated program id
+[programs.localnet]
+gateway_v2 = "FSgDgZoNxiUarRWJYrMDWcsZycNyEXaME5i3ZXPnhrWe"
+[programs.devnet]
+gateway_v2 = "FSgDgZoNxiUarRWJYrMDWcsZycNyEXaME5i3ZXPnhrWe"
+[programs.mainnet]
+gateway_v2 = "FSgDgZoNxiUarRWJYrMDWcsZycNyEXaME5i3ZXPnhrWe"
+
+[registry]
+url = "https://anchor.projectserum.com"
+
+[provider]
+cluster = "localnet"
+wallet = "./tests/fixtures/keypairs/7aHML6LXPtEuUDu4Azv52KbgEqdSDk9syqFvCAE63ZT5.json"
```

Comment:
Do we need this? I added this in copy/paste style - don't think it is needed... let's confirm

Reply:
Tried to build with it removed... "Error: Unable to deserialize config: missing field `provider` at line 22 column 1"